reference/conanfile/tools/cmake/cmaketoolchain.rst

```diff
@@ -241,25 +247,36 @@ With the ``CMakeToolchain`` it is possible to do, for multi-configuration system
 ``-G "Visual Studio 15"`` instead of the ``-G "Visual Studio 15 Win64"``
 
 
-For single-configuration build systems:
+If you are using a single-configuration generator:
 
 .. code:: bash
 
-    # Lets start in the folder containing the conanfile.py
-    $ mkdir build_release && cd build_release
-    $ conan install ..
-    # the build type Release is encoded in the toolchain already.
-    # This conan_toolchain.cmake is specific for release
-    $ cmake .. -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake
-    $ cmake --build .  # or just "make"
+    $ cmake ..  -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release
+    $ cmake --build
+
+
+It is recommended to use the ``cmake_layout(self)`` in the ``layout()`` method of your ``conanfile.py``. If a layout
+is declared, the ``CMakeUserPresets.json`` file will be generated in the same folder of your ``CMakeLists.txt`` file,
+so you can use the ``--preset`` argument from ``cmake >= 3.23`` or use an IDE:
+
+
+.. code:: bash
+
+    # The conan_toolchain.cmake is common for both configurations and will be located at "build/generators"
+    $ conan install .
+    $ conan install . -s build_type=Debug
+
+    # For single-configuration generator
+    $ cmake --preset Debug
+    $ cmake --build --preset "Build Debug"
+    $ cmake --preset Release
+    $ cmake --build --preset "Build Release"
+
+    # For multi-configuration generator
+    $ cmake --preset default
+    $ cmake --build --preset "Build Debug"
+    $ cmake --build --preset "Build Release"
```

Comment:
Shouldn't all of these be `--preset Debug` `--preset Release` ?

Reply:
The name of the preset is "Build XXX" but I think it should be ok to change it at: https://github.com/conan-io/conan/pull/11117
Will try.